package/docker/bahmni-reports/Dockerfile

```diff
@@ -22,7 +24,10 @@ COPY target/bahmnireports.war /etc/bahmni-reports/bahmnireports.war
 RUN cd ${WAR_DIRECTORY} && jar xvf /etc/bahmni-reports/bahmnireports.war
 
 ADD https://repo.mybahmni.org/packages/build/bahmni-embedded-tomcat-8.0.42.jar /opt/bahmni-reports/lib/bahmni-embedded-tomcat.jar
+RUN cd /opt/ && git clone https://github.com/Bahmni/bahmni-reports-extension-icd10.git
+RUN cd /opt/bahmni-reports-extension-icd10 && mvn clean install
```

Comment:
you can remove the directory `/opt/bahmni-reports-extension-icd10` after copying of jar file